applications/nrf_desktop/README.rst

```diff
@@ -1085,46 +1127,58 @@ Optionally, you can also enable the following module:
 
 .. _nrf_desktop_bootloader:
 
-Bootloader and DFU
-==================
+Bootloader
+==========
+
+The nRF Desktop application uses Secure Bootloader, referred in this documentation as B0.
 
-The nRF Desktop application uses Secure Bootloader, referred in this documentation as the B0.
+B0 is a small, simple, and secure bootloader that allows the application to boot directly from one of the application slots, thus increasing the speed of the direct firmware upgrade (DFU) process.
+More information about the B0 can be found at the :ref:`bootloader` page.
+
+.. note::
+    The nRF Desktop application does not support :ref:`MCUboot <mcuboot:mcuboot_wrapper>` as a second stage bootloader.
+
+    However, MCUboot can be enabled instead of B0.
+    Although the nRF Desktop application does not use MCUboot for the :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`, it can be the preferred choice for devices where the :ref:`USB serial recovery <nrf_desktop_bootloader_serial_dfu>` is to be used instead (for example, due to flash memory size limitations).
 
-The B0 is a small, simple and secure bootloader that allows the application to boot directly from one of the application slots, thus increasing the speed of the direct firmware upgrade (DFU) process.
-More information about the B0 can be found at :ref:`bootloader`.
 
 Enabling the bootloader
-    To enable the B0 bootloader, select the :option:`CONFIG_SECURE_BOOT` Kconfig option.
-    If this option is not selected, the application will be built without the bootloader, and the DFU will not be available.
+-----------------------
+
+To enable the B0 bootloader, select the :option:`CONFIG_SECURE_BOOT` Kconfig option.
+If this option is not selected, the application will be built without the bootloader, and the background DFU will not be available.
```

Comment:
`without the B0 bootloader` (there is also MCUBoot)

Reply:
Fixed.